src/pages/home/report/ReportActionsView.js

```diff
@@ -65,23 +64,22 @@ class ReportActionsView extends React.Component {
         this.didSubscribeToReportTypingEvents = false;
 
         this.unsubscribeVisibilityListener = null;
+        this.sortedReportActions = this.getSortedReportActionsForDisplay(props.reportActions);
 
         this.state = {
             isFloatingMessageCounterVisible: false,
-            newMarkerSequenceNumber: ReportUtils.isUnread(props.report)
-                ? props.report.lastReadSequenceNumber + 1
-                : 0,
+            newMarkerReportActionID: this.computeNewMarkerReportActionID(),
         };
 
         this.currentScrollOffset = 0;
-        this.sortedReportActions = this.getSortedReportActionsForDisplay(props.reportActions);
         this.mostRecentIOUReportActionID = ReportActionsUtils.getMostRecentIOUReportActionID(props.reportActions);
         this.trackScroll = this.trackScroll.bind(this);
         this.toggleFloatingMessageCounter = this.toggleFloatingMessageCounter.bind(this);
         this.loadMoreChats = this.loadMoreChats.bind(this);
         this.recordTimeToMeasureItemLayout = this.recordTimeToMeasureItemLayout.bind(this);
         this.scrollToBottomAndMarkReportAsRead = this.scrollToBottomAndMarkReportAsRead.bind(this);
         this.openReportIfNecessary = this.openReportIfNecessary.bind(this);
+        this.computeNewMarkerReportActionID = this.computeNewMarkerReportActionID.bind(this);
```

Comment:
This does not need to be bound to `this` because it's not passed to a child component and it's always executed in the context in which it's declared (the class in this case)

Reply:
Right you are! Thanks.